### ISSUE TITLE
Add support for shared notebooks

### ISSUE THREAD

**Issue by dominikmayer** [HappenApps/Quiver-iOS#13]
I have a shared notebook on our company Box and would like to add that to iOS. I know that Box is not yet supported but I do also not see how I would add the shared notebook if it was on another service. Would it be by adding a second service?

**Comment by ylian** [comment]
Right, I haven't got around to shared notebooks. It's trivial to add. Box will take some work, but also not too hard.
